extension/ts/content/adapters/tags/parseTags.ts

```diff
@@ -14,7 +14,7 @@ const parseRows = ({rows, fromRow, depth, tree, parent}: ParserOptions): number
 		const tag = rows[row][depth];
 		if (tag) {
 			const node = {tag, parent};
-			tree.set(tag, node);
+			tree.set(tag.toLowerCase(), node);
```

Comment:
are the keys all the lowercase tags and the values the actual tags from the sheet (with correct casing)?

Reply:
yeah i should probably write that somewhere